.betterer.results

```diff
@@ -5984,7 +5984,9 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "2"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "3"],
-      [0, 0, 0, "Unexpected any. Specify a different type.", "4"]
+      [0, 0, 0, "Unexpected any. Specify a different type.", "4"],
```

Comment:
Ok so here's my formal apology for this;

The way the form renderer for cloud receivers is implemented makes it sort of impossible to know what `value` type we'll actually receive – this depends on what `element` is chose – regular input fields yield `string` but checkboxes yield `boolean`.

I could probably have used `unknown` but this forces type casting and is also discouraged by Betterer /shrug 

If anyone has a good suggestion that doesn't require rewriting the entire nested form renderer I'm more than happy to put in the work :)

Reply:
Could we type it with the types we know today? Such as `string | boolean`